element_electrode_localization/coordinate_framework.py

```diff
@@ -56,13 +71,124 @@ class Voxel(dj.Part):
         -> CCF.Voxel
         """
 
+    @classmethod
+    def retrieve_acronym(self, acronym):
+        """ Retrieve the DataJoint translation of the CCF acronym"""
+        return re.sub(r'(?<!^)(?=[A-Z])', '_', acronym).lower()
```

Comment:
My fault for adding duplicative code during our April sprint. Separate this as helper function that can be referenced both here and in `load_ccf_annotation`?

Reply:
No problem @CBroz1!  We can refactor in the next phase.  Thanks.

Reply:
https://github.com/datajoint/element-electrode-localization/issues/10